build.gradle

```diff
@@ -16,10 +20,20 @@ dependencies {
     compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
 
     implementation 'org.eclipse.jgit:org.eclipse.jgit:6.4.0.202211300538-r'
+    implementation 'org.jetbrains:annotations:13.0'
```

Comment:
can we remove this dependency and use Nonnull instead ? (as Duarte PR)

Reply:
fix in https://github.com/SonarSource/git-files-blame/commit/d82887f2616f10ac140e472c707ef5eb35efca69